lib/Stats.js

```diff
@@ -460,20 +471,29 @@ Stats.jsonToString = function jsonToString(obj, useColors) {
 		colors.bold(obj.publicPath);
 		newline();
 	}
+
 	if(obj.assets && obj.assets.length > 0) {
 		var t = [
-			["Asset", "Size", "Chunks", "", "Chunk Names"]
+			[
+				{value: "Asset", color: colors.bold},
+				{value: "Size", color: colors.bold},
+				{value: "Chunks", color: colors.bold},
+				{value: "", color: colors.bold},
+				{value: "", color: colors.bold},
```

Comment:
:+1: but sadly this changes the stats output in a weirds way:

``` diff
     Hash: 8bac53cc0c37d71baf6b
       Time: Xms
      -    Asset     Size  Chunks                  Chunk Names
      -bundle.js  7.22 kB       0  [emitted]       main
      +    Asset     Size  Chunks             Chunk Names
      +bundle.js  7.22 kB       0  [emitted]  main
```

Could you make sure that empty columns doesn't need size? A simple check of `colSizes[col] !== 0` in `table` above would do that.